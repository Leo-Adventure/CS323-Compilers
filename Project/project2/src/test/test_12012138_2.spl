struct rabbit{
    int a;
    char c;
};
int main(){
    struct rabbit rrr;
    int ra = rrr.a;
    char rc = rrr.c;
    int error = ra && rc;
    error = error + 1;
    rrr = rrr + 114514;
    error = ra[1]+1;
    return 0;
}