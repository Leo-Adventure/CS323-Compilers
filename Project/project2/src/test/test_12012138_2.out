Error type 7 at line 9: binary operation on non-number variables
Error type 5 at line 9: unmatching type on both sides of assignment
Error type 7 at line 11: binary operation on non-number variables
Error type 5 at line 11: unmatching type on both sides of assignment
Error type 10 at line 12: indexing on non-array variable
Error type 7 at line 12: binary operation on non-number variables
Error type 5 at line 12: unmatching type on both sides of assignment